nilearn/_utils/docs.py

```diff
@@ -148,25 +147,21 @@ def custom_function(vertices):
         - `svc_l1`: :class:`{svc} <sklearn.svm.LinearSVC>` with L1 penalty.
             .. code-block:: python
 
-                svc_l1 = LinearSVC(penalty="l1",
-                                   dual=False,
-                                   max_iter=1e4)
+                svc_l1 = LinearSVC(penalty="l1", dual=False, max_iter=1e4)
 
         - `logistic`: \
             :class:`{logistic} <sklearn.linear_model.LogisticRegressionCV>` \
             with L2 penalty.
             .. code-block:: python
 
-                logistic = LogisticRegression(penalty="l2",
-                                              solver="liblinear")
+                logistic = LogisticRegression(penalty="l2", solver="liblinear")
 
         - `logistic_l1`: \
             :class:`{logistic} <sklearn.linear_model.LogisticRegressionCV>` \
             with L1 penalty.
             .. code-block:: python
 
-                logistic_l1 = LogisticRegression(penalty="l1",
-                                                 solver="liblinear")
+                logistic_l1 = LogisticRegression(penalty="l1", solver="liblinear")  # noqa
```

Comment:
Am not happy about having to use a noqa here but without it flake8 complains about line length and blacken docs will reformat any line break.

Reply:
Will merge and open an issue to remember to find a fix for this minor detail